### PR TITLE
Countdown timer

### PR DESCRIPTION
**Pull request by Amulya301**
Team,
Here are the files related to the timer tab. Please do review it and mention if any changes have to be made.

Thanks.

### CODE REVIEW

**Review by GcvKishore — APPROVED** [review]
All the files are updated as per the requirement.
Thanks.

File: website/templates/website/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			<form class="flex-w flex-c-m contact100-form validate-form p-t-70">
+				<div class="wrap-input100 validate-input where1" data-validate = "Email is required: ex@abc.xyz">
+					<input class="s1-txt1 placeholder0 input100" type="text" name="email" placeholder="Email Address">
+					<span class="focus-input100"></span>
+				</div>
+
+				<button class="flex-c-m s1-txt1 size2 how-btn trans-04 where1">
+					Notify Me
+				</button>
+			</form>
```
Comment (Vedvund) [inline website/templates/website/index.html]: As the form doesn't store any email id or doesn't have any functionality. I guess we can drop it for now...

**Review by Vedvund — CHANGES_REQUESTED** [review]
I felt it's a lot of CSS for a simple countdown page...
and maybe the bg01.jpg file can also be dropped because it is not been used in our project...

Please do check the requirements.txt as well because we strongly decided not to add the packages which we didn't use in the project. (optional change. it is not required for now but would be good practice)

I have attached the screenshot showing the view in my system. (only for confirmation)
Screenshot from 2022-04-03 00-37-15

**Review by GcvKishore — APPROVED** [review]

**Review by GcvKishore — APPROVED** [review]

**Review by Vedvund — APPROVED** [review]
The code looks relevant and the website works all discussed

**Review by priyushasunkara — APPROVED** [review]
Looks good.

**Pull request merged**